### PR TITLE
Add poller_tag test and remove copy_shell in action because not need …

### PR DESCRIPTION
**Pull request by ddurieux**
…it...

-----

The copy_shell is not necessary because we serialize the list of checks/actions before sending it to poller, so create a new check/action with same properties before send is not required

### CODE REVIEW

File: alignak/dispatcher.py [hunk 1]
```diff
@@ -531,7 +531,7 @@ def prepare_dispatch_other_satellites(self, sat_type, realm, cfg, arbiters_cfg):
         satellite_string = "[%s] Dispatching %s satellites ordered as: " % (
             realm.get_name(), sat_type)
         for sat in satellites:
-            satellite_string += '%s (spare:%s), ' % (
+            satellite_string += '%s (e:%s), ' % (
```
Comment (mohierf) [inline alignak/dispatcher.py]: Why removing the 'spare' information here?

File: test/test_poller_tag.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        self.print_header()
+        host = self._sched.hosts.find_by_name("test_host_pt_02")
+        self.external_command_loop()
+        checks = self.schedulers['scheduler-master'].sched.checks.values()
+        mycheck = self._sched.checks[host.checks_in_progress[0]]
+        assert mycheck.poller_tag == 'south'
+
+    def test_poller_tag_host_command(self):
+        """
+        We have a command with poller_tag: north
```
Comment (mohierf) [inline test/test_poller_tag.py]: Test description is better on the one and first line ... this for py.test to display a clean test name

File: test/test_poller_tag.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        svc.checks_in_progress = []
+        svc.act_depend_of = []
+        self.external_command_loop()
+        checks = self.schedulers['scheduler-master'].sched.checks.values()
+        mycheck = self._sched.checks[svc.checks_in_progress[0]]
+        assert mycheck.poller_tag == 'north'
+
+    def test_poller_tag_service_command(self):
+        """
+        We have a service with a poller_tag: south
```
Comment (mohierf) [inline test/test_poller_tag.py]: Same as previous ...

File: test/test_poller_tag.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+        svc.checks_in_progress = []
+        svc.act_depend_of = []
+        self.external_command_loop()
+        checks = self.schedulers['scheduler-master'].sched.checks.values()
+        mycheck = self._sched.checks[svc.checks_in_progress[0]]
+        assert mycheck.poller_tag == 'south'
+
+    def test_poller_tag_service_host(self):
+        """
+        We have a service with a poller_tag: north
```
Comment (mohierf) [inline test/test_poller_tag.py]: Same...

**Comment by mohierf** [comment]
May be interesting that you explain why the copy_shell is not necessary: in the PR comment please for that we know what it is about in the project history.

-----

This PR should be merged after the #603 because the removing of copy_shell potential impact will be detected! Merge #603 before I can merge this PR :wink:

File: alignak/dispatcher.py (hunk 1 above)
Comment (ddurieux) [inline alignak/dispatcher.py]: In really, I not remove, I rewritten the test from scratch

File: test/test_poller_tag.py (hunk 2 above)
Comment (ddurieux) [inline test/test_poller_tag.py]: ah ok

**Comment by ddurieux** [comment]
The copy_shell is not necessary because we serialize the list of checks/actions before send it to poller, so create a new check/action with same properties before send is not required

File: alignak/dispatcher.py (hunk 1 above)
Comment (ddurieux) [inline alignak/dispatcher.py]: Not remember I have modified that, I check

File: alignak/satellite.py [hunk 5]
```diff
@@ -702,12 +702,12 @@ def do_get_new_actions(self):
             # Ok, con is unknown, so we create it
             # Or maybe is the connection lost, we recreate it
             except (HTTPEXCEPTIONS, KeyError), exp:
-                logger.debug('get_new_actions exception:: %s,%s ', type(exp), str(exp))
+                logger.error('get_new_actions HTTP exception:: %s,%s ', type(exp), str(exp))
```
Comment (mohierf) [inline alignak/satellite.py]: Should use logger.exception:
```
logger.exception('get_new_actions HTTP exception:: %s', exp)
```

File: alignak/satellite.py [hunk 6]
```diff
@@ -702,12 +702,12 @@ def do_get_new_actions(self):
...
                 self.pynag_con_init(sched_id)
             # scheduler must not be initialized
             # or scheduler must not have checks
             except AttributeError, exp:
-                logger.debug('get_new_actions exception:: %s,%s ', type(exp), str(exp))
+                logger.error('get_new_actions attribute exception:: %s,%s ', type(exp), str(exp))
```
Comment (mohierf) [inline alignak/satellite.py]: Same comment as previous

**Pull request merged**